activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Add logging to report async queries lock wait duration
```

Comment:
Isn't it a bit weird to add a changelog entry for a modification of a feature that still isn't released? Maybe we shoud just edit the original async queries changelog entry?

Reply:
@byroot Thanks for taking a look! I completely missed #41372 🤦 

> Maybe we should just edit the original async queries changelog entry?

I've took another shot at ac3aab94f2 and included your suggestions below :pray: